pages/index.vue

```diff
@@ -58,90 +45,102 @@ export default Vue.extend({
       };
     }
 
-    if (homeSectionsArray) {
-      // Convert to an array
-      homeSectionsArray = Object.values(homeSectionsArray);
-
-      const homeSections = [];
-
-      for (const homeSection of homeSectionsArray as Array<string>) {
-        switch (homeSection) {
-          case 'librarytiles': {
-            homeSections.push({
-              name: this.$t('libraries'),
-              libraryId: '',
-              shape: 'thumb-card',
-              type: 'libraries'
-            });
-            break;
-          }
-          case 'latestmedia': {
-            const latestMediaSections = [];
-
-            const userViewsRequest = await this.$api.userViews.getUserViews({
-              userId: this.$auth.user?.Id
-            });
-
-            if (userViewsRequest.data.Items) {
-              const excludeViewTypes = [
-                'playlists',
-                'livetv',
-                'boxsets',
-                'channels'
-              ];
-
-              for (const userView of userViewsRequest.data.Items) {
-                if (
-                  excludeViewTypes.includes(userView.CollectionType as string)
-                ) {
-                  continue;
-                }
-
-                latestMediaSections.push({
-                  name: this.$t('latestLibrary', {
-                    libraryName: userView.Name
-                  }),
-                  libraryId: userView.Id || '',
-                  shape: getShapeFromCollectionType(userView.CollectionType),
-                  type: 'latestmedia'
-                });
+    // Convert to an array
+    homeSectionsArray = Object.values(homeSectionsArray);
+
+    const homeSections = [];
+
+    for (const homeSection of homeSectionsArray as Array<string>) {
+      switch (homeSection) {
+        case 'librarytiles': {
+          homeSections.push({
+            name: app.i18n.t('libraries'),
```

Comment:
This does not return any languages other than english

Reply:
Not tried it, but this should be due to the fact the returned value is stored, so not updated once locales are loaded

Reply:
It does, but you have to leave the page and come back.

Issue is that asyncData is generating the object that is then merged with data. As far as I can tell, there's no real way to fix this unless we refactor the way this page works (which is probably outside the scope of this).

Once we move the language selector to the settings, this shouldn't be an issue, however, so maybe we can bump up the priority on that.

Reply:
Ah I though I did that, either way it works now 👍